fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -83,16 +84,20 @@
 
     @Override
     public void initialize(final NamespaceRegistry registry,
-                           final NodeTypeManager nodeTypeManager) throws RepositoryException, IOException {
-        super.initialize(registry, nodeTypeManager);
+                           final NodeTypeManager nodeTypeManager) throws IOException {
+        try {
+            super.initialize(registry, nodeTypeManager);
+        } catch (RepositoryException e) {
+            throw new RepositoryRuntimeException("Error initializing FedoraFileSystemConnector!", e);
```

Comment:
No problem with this code, just a note to ourselves: we do so much `throw new RepositoryRuntimeException(something)` that I'm starting to wonder if we shouldn't write some static methods to the purpose a la Guavas `Throwables.propagate(...)`.